src/field/pokemon.ts

```diff
@@ -2212,7 +2210,8 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     }
     const move = new PokemonMove(moveId);
     this.moveset[moveIndex] = move;
-    if (this.summonData?.moveset) {
+    // TODO: should this also be modifying the summon data moveset?
+    if (this.summonData.moveset.length) {
```

Comment:
Not length check shortcuts please

```suggestion
    if (this.summonData.moveset.length !== 0) {
```